File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        unique = set(nums)
        longest = 0
        for item in unique:
            if (item - 1) not in unique:
                length = 1
                while (item + length) in unique:
                    length = length + 1
                longest = max(longest, length)
        return longest



        

        

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        HashMap<Integer, Integer> map = new HashMap<>();

        int[] arr = new int[k];

        for(int i = 0; i < nums.length; i++) {
            map.put(nums[i], map.getOrDefault(nums[i], 0) + 1);
        }

        for (int i = 0; i < k; i++) {
            arr[i] = Collections.max(map.entrySet(), Map.Entry.comparingByValue()).getKey();
            map.put(arr[i], 0);
        }

        return arr;
    }
}
